### ISSUE TITLE
orgc_id column not indexed on 2.4.130

### ISSUE THREAD

**Issue by BurakOzgen** [MISP/MISP#6229]
# orgc_id indexing problem on 2.4.130

After updating .129 to .130 this problem occured.

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 18.04
| PHP version               | 7.2.24
| MISP version / git hash   | 2.4.130, 7d253f87fdc452a1224097cbaa80078caf46e141
| Browser                   | Chrome

### Logs, screenshots, configuration dump, ...
![image](https://user-images.githubusercontent.com/49974892/91029053-f7372500-e606-11ea-9338-096eb20e76e0.png)

**Comment by seldem** [comment]
Same here

**Comment by mokaddem** [comment]
After some investigation I found the culprit.

- You can fix it by either rolling back to an older db version then letting MISP update itself **[slower]**
```sql
update admin_settings set value=50 where setting='db_version';
```
- or by creating the index manually with the command provided by clicking on the wrench **[faster]**
```sql
CREATE INDEX `orgc_id` ON `feeds` (`orgc_id`);
```

Thanks for the report!

**Comment by mokaddem** [comment]
Fixed with 355861d
